src/Coccinelle-for-Rust.md

```diff
@@ -41,7 +41,18 @@ flow is not yet supported.
 
 [Coccinelle for Rust](https://gitlab.inria.fr/coccinelle/coccinelleforrust.git)
 
-[A recent talk about Coccinelle for Rust](https://gitlab.inria.fr/coccinelle/coccinelleforrust/-/blob/main/talks/rfl.pdf)
+[LWN article from Kangregos 2024](https://lwn.net/Articles/991399/)
+
+[A recent talk about Coccinelle for Rust](https://gitlab.inria.fr/coccinelle/coccinelleforrust/-/raw/ctl2/talks/lpc24.pdf?ref_type=heads)
+
+## Feedback
+
+Coccinelle For Rust relies on user feedback for its improvement.
+Several bugs have been fixed thanks to helpful feedback from users. 
+If you are interested in using Coccinelle For Rust please feel free to
+reach out to us at Julia.Lawall@inria.fr and tathagata.roy1278@gmail.com
+with your questions or feedback.
```

Comment:
A suggestion (of course, please feel free to ignore!): the contact emails are at the bottom as well, perhaps you could refer to that section (with a link) to avoid duplication. The disadvantage is that may make it harder for people to see, i.e. perhaps you prefer to have the emails directly here so that they can click them.

Orthogonality, the emails at the bottom use a link with `mailto:`. I think they do not get automatically linked with this software (unlike in the preview in GitHub), so you may want to double-check the generation.

Reply:
> A suggestion (of course, please feel free to ignore!): the contact emails are at the bottom as well, perhaps you could refer to that section (with a link) to avoid duplication. The disadvantage is that may make it harder for people to see, i.e. perhaps you prefer to have the emails directly here so that they can click them.

Thank you! I changed it to add a link.

> Orthogonality, the emails at the bottom use a link with mailto:. I think they do not get automatically linked with this software (unlike in the preview in GitHub), so you may want to double-check the generation.

I am sorry but, I think I do not understand what you mean.
Is it okay with the changes I made?

Thank you so much!

Reply:
You're welcome! I meant that, if you decided to avoid the "Contact" link, then those emails were probably not going to be rendered as links unless you used:

    [...@...](mailto:...@...)

i.e. like it is done in the "Contact" section for the emails.

Reply:
Oh I see. Thank you for the explanation!

Reply:
You're welcome!

I fixed the link (the anchor/`id` needed to be lowercase to work) and published it -- it is now live, please take a look: https://rust-for-linux.com/coccinelle-for-rust

Reply:
Thank you so much!